src/tests/GC/Regressions/Github/Runtime_76219/Runtime_76219.csproj

```diff
@@ -2,23 +2,14 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <Optimize>True</Optimize>
-
-    <!-- This test needs GCStress mode but it doesn't take long to execute
-         so is fine to keep it in the inner loop -->
-    <CLRTestBatchPreCommands><![CDATA[
-      $(CLRTestBatchPreCommands)
-      set DOTNET_TieredCompilation=0
-      set DOTNET_GCStress=0x3
-      set DOTNET_COMPlus_GCSegmentSize=8000
-      ]]></CLRTestBatchPreCommands>
-          <BashCLRTestPreCommands><![CDATA[
-      $(BashCLRTestPreCommands)
-      export DOTNET_TieredCompilation=0
-      export DOTNET_GCStress=0x3
-      export DOTNET_COMPlus_GCSegmentSize=8000
-      ]]></BashCLRTestPreCommands>
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="$(MSBuildProjectName).cs" />
+
+    <!-- This test needs GCStress mode but it doesn't take long to execute
+         so is fine to keep it in the inner loop -->
+    <CLRTestEnvironmentVariable Include="DOTNET_TieredCompilation" Value="0" />
+    <CLRTestEnvironmentVariable Include="DOTNET_GCStress" Value="0x3" />
+    <CLRTestEnvironmentVariable Include="DOTNET_COMPlus_GCSegmentSize" Value="8000" />
```

Comment:
@EgorBo I assume that "DOTNET_COMPlus" was a typo, but should it be fixed or removed?

Reply:
Fixed by #76997